test/powershell/Modules/Microsoft.PowerShell.Core/Invoke-Command.Tests.ps1

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe "Invoke-Command" -Tags "CI" {
+    Context "StrictModeVersion tests" {
+        BeforeAll {
+            if ([ExperimentalFeature]::IsEnabled("PSStrictModeVersionAssignment")){
+                $skipTest = $false
+            }
+            else {
+                $skipTest = $true
+            }
+            $errorMessage = "The variable '`$InvokeCommand__Test' cannot be retrieved because it has not been set."
+            If (Test-Path Variable:InvokeCommand__Test) {
+                Remove-Item Variable:InvokeCommand__Test
+            }
+        }
+
+        It "Setting -StrictModeVersion parameter with uninitialized variable throws error"  -skip:$skipTest {
+            { Invoke-Command -StrictModeVersion 3.0 {$InvokeCommand__Test} } | Should -Throw $errorMessage
```

Comment:
Since error messages will eventually be localized to other languages, we can't use them for testing success, since tests will start failing when run in other locales.  Instead, we use the ErrorRecord `FullyQualifiedErrorId` property which is never localized.  For example:

```powershell
{ Invoke-Command -SetrictModeVersion 3.0 { $InvokeCommand_Test } | Should -Throw -ErrorId 'VariableIsUndefined'
```